index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Rominas Microverse Student Project 1</title>
+<link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <div class="logo">
+            <a href="#">My logo</a>
+            </div>
+        <img class="menu" src="images/Union.svg" alt="menu">
+        </div>
+    </header>
+<section class="my-name">
+    <div class="name"><h1>I'm Romina! <br>Glad to see<br> you</h1></div> 
+    <div class="greetings"><p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="connect"> LET'S CONNECT</div>
+    <div class="social-media">
+        <ul>
+            <li><img class="social-media twitter" src="images/twitter.svg" alt="twitter" /></li>  
+            <li><img class="social-media linkedin" src="images/linkedin.svg" alt="linkedin"/></li>
+            <li><img class="social-media github" src="images/github.svg" alt="github" /></li>
+            <li> <img class="social-media medium" src="images/medium.svg" alt="medium" /></li>
+            <li><img class="social-media medium" src="images/vector.svg" alt="vector" /></li>
```

Comment:
- [OPTIONAL] I strongly recommend using `a` tags inside the `li` tags to make the icons clickable as this is a good way to redirect the user to the links.